src/components/MakeWidget/MakeWidget.styles.tsx

```diff
@@ -3,23 +3,36 @@ import styled from "styled-components/macro";
 import AddressInput from "./subcomponents/AddressInput/AddressInput";
 import InfoSection from "./subcomponents/InfoSection/InfoSection";
 import OrderTypeSelector from "./subcomponents/OrderTypeSelector/OrderTypeSelector";
+import { RateField } from "./subcomponents/RateField/RateField";
 
 export const Container = styled.div`
   display: flex;
   flex-direction: column;
-  height: 100%;
+  flex-grow: 1;
 `;
 
 export const StyledOrderTypeSelector = styled(OrderTypeSelector)`
+  margin-bottom: 1rem;
+`;
+
+export const StyledRateField = styled(RateField)`
+  margin-left: 1rem;
+  margin-bottom: 1rem;
+`;
+
+export const OrderTypeSelectorAndRateFieldWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
```

Comment:
I've been thinking about this one. I agree, but we want space-between on desktop and there is no way to see if the element has wrapped with only css, so we would need a javascript solution which is not ideal.

It would be a little bit easier if both elements had a fixed width, but they're both dynamic (how long will "specific wallet" be in other languages?) making it even more complicated.

So maybe we should skip this one unless you have a better solution.

Reply:
From what I can see the maximum pixel difference when they are unwrapped with ```space-around``` would be this:

````space-between```` with maximum gap
<img width="432" alt="Screen Shot 2022-08-20 at 4 35 31 PM" src="https://user-images.githubusercontent.com/80134686/185765172-3981d75a-9553-4688-8a76-b1653084ff8d.png">

````space-around```` with maximum gap
<img width="429" alt="Screen Shot 2022-08-20 at 4 35 16 PM" src="https://user-images.githubusercontent.com/80134686/185765180-e50ba409-58f7-4b07-816c-a31454ab180e.png">

in my opinion that pixel difference is less worrisome than the left alignment which happens under ````space-between```` when it responds to wrap. does that make sense? in the end, it is not a huge issue though either way.

Reply:
Makes sense but your theoretical maximum gap is with the english language and a "1" exchange rate. It could be way different depending on the situation.

I fixed it by getting rid of flex-wrap and using a media query for `phoneOnly`. Hope this works for now.

Reply:
I think that is a good fix. I added a change to keep the RateBox styling clean under edge cases.